packages/x-components/src/x-modules/facets/wiring.ts

```diff
@@ -110,6 +124,12 @@ export const facetsWiring = createWiring({
   FacetsGroupProvided: {
     setFacetsGroupWire
   },
+  UserAcceptedAQuery: {
+    setQuery
+  },
+  FacetsQueryChanged: {
+    clearFiltersOnStatusWire
```

Comment:
Maybe we can get rid of the status prop in the module if we clear the filters only if the facets had already a query.

To know that we can get the old value accessing the metadata argument. This is useful to mantain the pre-selected filters.

```suggestion
    clearAllFilters: filter(clearAllFiltersWire, ({ metadata }) => !!metadata.oldValue)
```

Reply:
good catch! I think that it is a better solution